src/io/flutter/inspections/SdkConfigurationNotificationProvider.java

```diff
@@ -108,6 +117,21 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
     return null;
   }
 
+  private EditorNotificationPanel createOutOfDateFlutterSdkPanel() {
+
+    final FlutterSettings settings = FlutterSettings.getInstance(project);
+    if (settings == null || settings.ignoreOutOfDateSdks()) return null;
+
+    EditorNotificationPanel panel = new EditorNotificationPanel();
+    panel.setText(FlutterBundle.message("flutter.old.sdk.warning"));
+    panel.createActionLabel("Dismiss", () -> {
+      settings.setIgnoreOutOfDateSdks(true);
```

Comment:
You could add something to the Flutter preferences page until the automagical event is devised.